[LOG] [E] [2021/07/14 07:14:13]: DatabaseManager.execute() Error: Invalid default value for 'created_at'
